python_bindings/src/halide/__init__.py

```diff
@@ -0,0 +1,7 @@
+import sys
+from pathlib import Path
+
+sys.path.append(str(Path(__file__).parent.resolve(strict=True)))
```

Comment:
A comment on this line would be nice.

Reply:
Yeah, this is a stop-gap.

I would really like this file to be just the following:
```
from .halide_ import *
from .halide_ import _, _1, ..., _9
```
However,
1. relative imports cannot be overloaded with `sys.path`
2. for a variety of reasons, copying the python sources next to the `halide_.*.so` module is difficult to make work in 100% of cases in CMake
3. even if we could copy them reliably, copying these files into the build folder seems inelegant

Fortunately, there are apparently other hooks besides `sys.path` that we could use to redirect a failing relative import. I haven't looked into them yet, but didn't feel like holding up this PR for it.

Reply:
Will do